src/codec/frame/mod.rs

```diff
@@ -243,6 +252,7 @@ impl FrameReadState {
         frame
     }
 
+    #[cfg(any(feature = "sync", feature = "async"))]
     fn merge_frame(&mut self, checked_frame: OwnedFrame) -> Result<Option<OwnedFrame>, WsError> {
```

Comment:
this function may be ok to be `pub`

Reply:
This is good to know! At present the preference to allow dead code is not actionable because these fields/methods are not public to parsers who could otherwise make use of them.

I will take another stab at this PR with pub visibility modifiers on the methods. Would you like to hide them from docs, or is it expected that users will want to use them?

Reply:
> 

hide them from docs looks ok for me.